.github/workflows/check-links.yaml

```diff
@@ -0,0 +1,34 @@
+name: Links
+
+on:
+
+  push:
+    paths:
+      - '.github/workflows/check-links.yaml'
+      - '.lycheeignore'
+
+  repository_dispatch:
+
+  workflow_dispatch:
+
+  schedule:
+    - cron: "00 09 * * *"
+
+jobs:
+  check_links:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v2
+
+      - name: Create Issue From File
```

Comment:
Let's skip creating issues, we can just use the checker on PRs

Reply:
Ok. I removed this step for tickets